build-conf/datasets.properties

```diff
@@ -69,8 +69,8 @@ SEQAMOD=
 # Optional IDz Load Library. Example: FEL.V14R0M0.SFELLOAD
 SFELLOAD=
 
-# Optional IDZ zUnit / WAZI VTP library containing necessary copybooks. Example : FEL.V14R2.SBZUSAMP
-SBZUSAMP=
+# Optional IBM Debug Tool library necessary TAZ copybooks. Example : DEBUG.V16.SEQASAMP
```

Comment:
Is this value optional?

Reply:
Generally it is optional, but obviously required when the user is leveraging TAZ